uber/templates/registration/discount.html

```diff
@@ -8,6 +8,9 @@
 {% csrf_token %}
 <table class="form">
 <tr>
+    <!-- Note that this is preferred first and last name.
+    May want to change, probably by sending an attendee ID to this page as a hidden field/something,
+    instead of having to re-search for them with potentially ambiguous/problematic results. -->
```

Comment:
Yeah, it's creating a new attendee.

It's actually very sparsely used, since we can now edit the 'base price' on any attendee's details page to get the same result, but I believe it's used at promo tables.